packages/veritone-react-common/src/components/IngestionAdapters/DynamicAdapter/index.js

```diff
@@ -102,7 +101,9 @@ class DynamicAdapter extends React.Component {
             </div>
           </div>
           <div>
-            <div className={styles.adapterContainer + ' ' + styles.flexContainer}>
+            <div
+              className={styles.adapterContainer + ' ' + styles.flexContainer}
```

Comment:
use https://www.npmjs.com/package/classnames you can "import classnames as cx from 'classnames'' then do replace this with "cx(styles.adapterContainer, styles.flexContainer)"